### PR TITLE
Update dot env

### PR DESCRIPTION
**Pull request by adamrmelnyk**
:elephant: 

* Updating dot-env and updating tests to stop using should.

### CODE REVIEW

**Comment by MathieuGilbert** [comment]
:book:

File: Gemfile [hunk 1]
```diff
@@ -5,4 +5,3 @@ gemspec
 
 gem 'rake'
 gem 'rspec'
-gem 'pimpmychangelog'
```
Comment (MathieuGilbert) [inline Gemfile]: :+1:

File: spec/whenever_spec.rb [hunk 2]
```diff
@@ -19,18 +19,19 @@
...
-      config.whenever_identifier.should == 'foo_bar'
+      expect(config.whenever_identifier).to eq 'foo_bar'
     end
   end
 
   describe 'whenever_cmd' do
     it 'has default value' do
-      config.whenever_cmd.should == 'whenever'
+      expect(config.whenever_cmd).to eq 'whenever'
+      expect(config.whenever_cmd).to eq 'whenever'
```
Comment (MathieuGilbert) [inline spec/whenever_spec.rb]: Duplicate?

**Comment by MathieuGilbert** [comment]
:hankey:

**Comment by adamrmelnyk** [comment]
:wave:

**Comment by jessedoyle** [comment]
:elephant: :elephant: :elephant:

**Comment by jbabe** [comment]
@adamrmelnyk do you remember if this went in for the Gemnasium updates?

**Pull request merged**